### PR TITLE
new icon: appwrite (plain, original, wordmark)

### PR DESCRIPTION
**Pull request by eldadfux**
I've added the original, plain, original-wormark, and plain-wordmark icons for https://appwrite.io/. Give it a check and let me know if there's something I should change.

### CODE REVIEW

**Review by amacado — CHANGES_REQUESTED** [review]
Icons looking good! :) Thanks for the contribution. Could you merge the devicon.json with our latest version? :)

**Comment by eldadfux** [comment]
Merged :)

File: devicon.json [hunk 1]
```diff
@@ -156,6 +157,24 @@
...
+            "plain",
+            "plain-wordmark"
+          ],
+          "font": [
+            "original",
+            "original-wordmark",
+            "plain",
+            "plain-wordmark"
+          ]
+        }
```
Comment (amacado) [inline devicon.json:176]: Adding the `color` for colored versions and aliases to not have the same icon twice in the font (only in css).

```suggestion
        },
        "color": "#f02e65",
        "aliases": [
            {
                "base": "plain",
                "alias": "original"
            },
            {
                "base": "plain-wordmark",
                "alias": "original-wordmark"
            }
        ]
```

**Review by amacado — CHANGES_REQUESTED** [review]
Icons looking good, but we do not them them twice in the set, see my code suggestions 😄 

new_icons

**Comment by Thomas-Boi** [comment]
Hello there,

We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues, and merge your icons ourselves.

Since GitHub tracks contributions by commits, you will still be credited. We also hope that you will help us merge this PR before the deadline comes.

Cheers,
Devicon Team

**Comment by eldadfux** [comment]
So sorry for the delays, its been a hectic few weeks. I will set myself a reminder to try and finish in in the next couple of weeks!

**Comment by Thomas-Boi** [comment]
Thank you for picking up this PR again! Just take your time and we really appreciate it 😄

**Comment by eldadfux** [comment]
Seems like I just needed to accept all the code suggestions? @amacado let me know if I missed something...

**Review by amacado — APPROVED** [review]
Sorry for the delay, been a little busy. Thanks for the update @eldadfux. appwrite.io will be in the upcomming devicon release! :)

**Comment by eldadfux** [comment]
Ohh this is awesome, we're so gonna tweet about it 🤟

**Comment by eldadfux** [comment]
Thanks guys!

image

**Pull request merged**